tapgarden/planter.go

```diff
@@ -135,6 +135,27 @@ type FinalizeParams struct {
 	SiblingTapTree fn.Option[asset.TapscriptTreeNodes]
 }
 
+// FundParams are the options available to change how a batch is funded, and how
+// the genesis TX is constructed.
+type FundParams struct {
+	FeeRate        fn.Option[chainfee.SatPerKWeight]
+	SiblingTapTree fn.Option[asset.TapscriptTreeNodes]
+	// TODO(jhb): follow-up PR: accept a PSBT here
+}
+
+// groupSeal specifies the group witness for a seedling in a funded batch.
+type groupSeal struct {
+	GroupMember  asset.ID
```

Comment:
Doesn't this also need the new asset name and the existing group key?

Reply:
At the point of providing witnesses, we can already display the asset ID for each seedling to the user, since the batch was already funded. And the info given when submitting the seedling is enough to map the seedling to the group it would be a part of (implemented in `buildGroupReqs`), so we don't need that info again.